Makefile

```diff
@@ -43,6 +44,17 @@ help: ## Show this help
 release: ## Create release binaries
 release: $(BINARY)-linux-amd64 $(BINARY)-darwin-amd64 $(BINARY).exe
 
+.PHONY: docker
+docker: ## Build and tag docker image
+docker: $(BINARY)-linux-amd64
+	docker build -t $(DOCKER_IMAGE):v$(VERSION) -t $(DOCKER_IMAGE):latest .
```

Comment:
Noob question: what about changing the version to: `VERSION=$(shell git describe --tags)`
from `VERSION=$(shell cat VERSION)`?

Just worried because I may want to create a temp version of my docker image... without overwriting the released version.

Reply:
I didn't know about that `git describe --tags`! That's pretty handy. I think in this case we want the short version number rather than one with the git hash when we're publishing to docker hub. We could add a separate build for a dev image if that's a common use-case. But for now if I wanted to retain the officially-released image and build a temporary image to test them side-by-side, I'd probably just do the build outside of the Makefile.

Reply:
Sounds good!!
This `git describe --tags` is usually used in git hooks to generate automatically the content of the `VERSION` file.